File: app/src/main/java/com/mcwilliams/TableTopicsApp/model/Member.java
package com.mcwilliams.TableTopicsApp.model;

/**
 * Created by m439047 on 5/29/13.
 */
public class Member {

    int _id;
    String _name;

    public Member() {
    }

    public Member(String name) {
        this._name = name;
    }

    public Member(int id, String name) {
        this._name = name;
        this._id = id;
    }

    public String get_name() {
        return _name;
    }

    public void set_name(String name) {
        this._name = name;
    }

    public int get_id() {
        return _id;
    }

    public void set_id(int _id) {
        this._id = _id;
    }
}

File: app/src/main/java/com/mcwilliams/TableTopicsApp/model/Member.kt
package com.mcwilliams.TableTopicsApp.model

import kotlin.properties.Delegates

/**
 * Created by m439047 on 5/29/13.
 */
class Member {

    var _id: Int = 0
    var _name: String by Delegates.notNull()

    constructor() {
    }

    constructor(name: String) {
        this._name = name
    }

    constructor(id: Int, name: String) {
        this._name = name
        this._id = id
    }
}
